doc/operations/troubleshooting.rst

```diff
@@ -0,0 +1,81 @@
+Troubleshooting CCF
+===================
+
+This page contains troubleshooting tips for CCF.
+
+Tips for interacting with CCF to diagnose issues
+------------------------------------------------
+.. note:: In the examples below this documentation uses ``example-ccf-domain.com`` as an example CCF domain, you will need to replace that with your own CCF domain when using these commands. 
+
+.. note:: CCF may be deployed with a loadbalancer which may cache the node which last responded to a query from an IP address. Until the cache clears, any subsequent queries from that IP address will be handled by the same node. As an example, if the cache clears after one minute, then in order to get a response from a different node, an operator must wait one minute between queries.  
+
+Below are descriptions of CLI commands and how they are useful for diagnosing CCF issues:
+
+**“What node is handling my requests?”**
+
+.. code-block:: bash 
+
+    curl -k https://example-ccf-domain.com/node/network/nodes/self -i
+
+This is useful to identify which node is handling queries. The node ID can be found in the Location header. Please read the note above about loadbalancers.
+
+**“What CCF version is running?”**
+
+.. code-block:: bash
+
+    curl -k https://example-ccf-domain.com/node/version
+
+This is useful to confirm the version that is running.
+
+**“What nodes are part of the current network?”**
+
+.. code-block:: bash
+
+    curl -k https://example-ccf-domain.com/node/network/nodes
+
+This will show information for all nodes in the network. In a healthy network all nodes will show ``“status” = “Trusted”``, and one node only will show ``“primary” = true``. This is the healthy state of the network. 
+Around upgrades/restarts/migrations nodes will transition through unhealthy states temporarily. If the network remains in an unhealthly state for a long time, this indicates there is an issue. 
+
+You can obtain this information for a single node by appending its ID to this endpoint. E.g. ``https://<your-domain>/node/network/nodes/<ID>``, where ``<ID>`` can be obtained from the ``/node/network/nodes/self`` endpoint described above. Take note of the ``node_data`` field which contains useful correlation IDs. E.g. ``node 0``, ``pod name``.
```

Comment:
When referring to endpoints, using the notation :http:GET:`/node/version` will create a link to the corresponding OpenAPI schema entry.

Reply:
Oh cool! I've now made use of this throughout